docs/snippets/common/storybook-preview-global-loader.js.mdx

```diff
@@ -5,7 +5,7 @@ import fetch from 'node-fetch';
 
 export const loaders = [
   async () => ({
-    currentUser: (await fetch('https://jsonplaceholder.typicode.com/users/1')).json(),
+    currentUser: await (await fetch('https://jsonplaceholder.typicode.com/todos/1')).json(),
```

Comment:
shouldn't this be fetching a user?

Reply:
Fixed with the latest commit. Marking it as solved